models/points.go

```diff
@@ -193,10 +193,18 @@ func parsePoint(buf []byte, defaultTime time.Time, precision string) (Point, err
 		return nil, err
 	}
 
+	// convert from slices to avoid buffer overwrites from services like UDP
+	keyBuf := make([]byte, len(key))
```

Comment:
Isn't this just introducing even more allocation? buf -> key -> keyBuf? Previously it was just buf -> key.